adcc/adc_pp/modified_transition_moments.py

```diff
@@ -60,6 +60,60 @@ def mtm_adc2(mp, dipop, intermediates):
     return AmplitudeVector(ph=f1, pphh=f2)
 
 
+def mtm_adc3(mp, dipop, intermediates):
+    second_order = mtm_adc2(mp, dipop, intermediates)
+    f1 = second_order.ph
+    f2 = second_order.pphh
+
+    t2 = mp.t2(b.oovv)
+    p0 = mp.mp2_diffdm
+    td2 = mp.td2(b.oovv)
+    tt2 = mp.tt2(b.ooovvv)
+    ts3 = mp.ts3(b.ov)
+    td3 = mp.td3(b.oovv)
+
+    f1 += (
+        + einsum('ib,ab->ia', ts3, dipop.vv)
+        - einsum('ja,ji->ia', ts3, dipop.oo)
+        - einsum('ijab,jb->ia', td3, dipop.ov)
+        + einsum('ijab,kb,jk->ia', t2, p0.ov, dipop.oo)
+        + einsum('jkab,kb,ji->ia', t2, p0.ov, dipop.oo)
+        - einsum('ijbc,jc,ab->ia', t2, p0.ov, dipop.vv)
+        - einsum('ijab,jc,cb->ia', t2, p0.ov, dipop.vv)
```

Comment:
Please add the `1.0` prefactors for consistency with the rest of the code block. This is whay we (hopefully) do throughout `adcc`.